src/collider/GeomTree.cpp

```diff
@@ -328,9 +286,14 @@ void GeomTree::Save(Serializer::Writer &wr) const
 	wr.Vector3d(m_aabb.min);
 	wr.Double(m_aabb.radius);
 
+	// TODO: the entire edge and triangle BVH should be written to disk and
+	// loaded in future SGM versions rather than being re-computed on each load
+
 	wr.Int32(m_numEdges);
 	for (Sint32 iAabb = 0; iAabb < m_numEdges; ++iAabb) {
-		wr << m_aabbs[iAabb];
+		AABBd &aabb = m_edgeAABBs.get()[iAabb];
+		// Write back an old-style min-max-radius Aabb for compatibility with old SGM versions
```

Comment:
Why do we need compatibility with the old SGM format, we update everything with every sneeze.

Reply:
I want to address several issues with an SGM format bump (vertex formats of stored geometry, for one), so I prefer here to simply maintain compatibility with the existing wire format and then completely rewrite the serialization code for GeomTree in a separate PR.

Essentially, this PR is concerned with addressing runtime performance and I'd rather not muddy the water with an overhaul of the SGM format right now.